Mention that header name in JWT configuration should be specified always (#72024)
grafana/grafana: docs/sources/setup-grafana/configure-security/configure-authentication/jwt/index.md

header_name = X-JWT-Assertion
```

{{% admonition type="warning" %}}

> header name should be specified even if only URL login is used, otherwise, the JWT token in the URL will be ignored.

Reviewer: ```suggestion
You must specify the header name even if only the URL login is used. Otherwise, the JWT token in the URL is ignored.
```
